doc/old/working_with_pixel_data.rst

```diff
@@ -140,3 +156,27 @@ transformations, the Modality LUT transformation is always applied first.
     ds = dcmread(fname)
     arr = ds.pixel_array
     hu = apply_modality_lut(arr, ds)
+
+
+VOI LUT or Windowing Operation
+------------------------------
+
+The DICOM :dcm:`VOI LUT<part03/sect_C.11.2.html>` module applies a
+VOI or windowing operation to input values. The
+:func:`~pydicom.pixel_data_handlers.util.apply_voi_lut` function
+can be used with an input array and a dataset containing a VOI LUT module to
+return values with applied VOI LUT or windowing. When a dataset contains
+multiple VOI or windowing views then a particular view can be returned by
+using the `index` keyword parameter.
+
+When a dataset requires multiple greyscale transformations, then its assumed
+that the modality LUT or rescale operation has already been applied.
+
+.. code-block:: python
+
+    from pydicom.pixel_data_handlers.util import apply_voi_lut
+
+    fname = get_testdata_files("")[0]
```

Comment:
Whoops

Reply:
Note too that there is a singular `get_testdata_file` now so don't need indexing.

Reply:
Done